airbyte-integrations/connectors/source-fastbill/source_fastbill/manifest.yaml

```diff
@@ -0,0 +1,116 @@
+version: "0.29.0"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: ["RESPONSE","{{ parameters.record_extractor }}"]
+  requester:
+    type: HttpRequester
+    url_base: "https://my.fastbill.com/api/1.0"
+    http_method: "POST"
+    authenticator:
+      class_name: source_fastbill.components.CustomAuthenticator
+      username: "config['username']"
+      password: "config['api_key']"
+    request_body_json:
```

Comment:
Can you also add the request header application/json here?

Reply:
Done added